soweego/target_selection/commons/utils/file_utils.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+# coding: utf-8
+
+import os
+import sys
+import json
+import datetime
+import codecs
+from shutil import copyfile
+from collections import namedtuple
+
+from .. import constants as const
+
+def json_deserialize(serialized_json: str) -> object:
+    return json.loads(serialized_json, object_hook=lambda d: namedtuple('X', d.keys())(*d.values()))
+
+def load_json(source_file: str) -> object:
+    try :
+        with open(source_file) as file:
+            return json.load(file)
+    except :
+        log_error('Unable to load .json file: {1}'.format(source_file)) 
+
+def log_error(text: str):
+    log('{0} \t {1} \n'.format(get_iso_time(), text), loc.log_file)
+
+def log(text: str, file: str, access_method = 'ab+'):
```

Comment:
Please do not redefine logging, use the already available one instead.
To enable logging in your Python file, just use:
```py
import logging

LOGGER = logging.getLogger(__name__)
```